backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobExecutionActor.{AbortedResponse, BackendJobExecutionResponse, ExecuteJobCommand, FailedNonRetryableResponse}
+import cromwell.backend.BackendLifecycleActor._
+
+import scala.concurrent.Future
+import scala.util.Try
+
+
+trait BackendCacheHitCopyingActor extends Actor with ActorLogging with BackendJobLifecycleActor {
+
+  def copyCachedOutputs: Future[BackendJobExecutionResponse]
+
+  def receive: Receive = LoggingReceive {
+    case ExecuteJobCommand =>
+      val shutdown: PartialFunction[Try[BackendJobExecutionResponse], Unit] = { case _ => context stop self }
+      performActionThenRespond(copyCachedOutputs andThen shutdown, onFailure = cachingFailed)
```

Comment:
For some reason the `PartialFunction` annotation looks very verbose. Could you do:
```
def doThenShutdown = copyCachedOutputs andThen { case _ => context.stop(self) }
performActionThenRespond(doThenShutdown, onFailure = cachingFailed)
```